src/System.IdentityModel.Tokens.Jwt/System.IdentityModel.Tokens.Jwt.csproj

```diff
@@ -26,5 +26,36 @@
     <ProjectReference Include="..\Microsoft.IdentityModel.JsonWebTokens\Microsoft.IdentityModel.JsonWebTokens.csproj" />
     <ProjectReference Include="..\Microsoft.IdentityModel.Tokens\Microsoft.IdentityModel.Tokens.csproj" />
   </ItemGroup>
+  
+   <ItemGroup>
+    <None Update="ClaimsKnowledge.tti">
+      <Generator></Generator>
+    </None>
+    <None Update="ClaimTypeAccessor.tt">
+      <LastGenOutput>ClaimTypeAccessor.gen.cs</LastGenOutput>
+      <Generator>TextTemplatingFileGenerator</Generator>
```

Comment:
When you save the ClaimTypeAccessor.tt this generates the ClaimTypeAccessor.gen.cs which contain a partial class implementation of ClaimTypeAccessor.cs